src/components/pages/__tests__/AFPStep.test.tsx

```diff
@@ -16,45 +16,45 @@ jest.mock('../../FormButtons', () => ({
   default: jest.fn(() => <div>Mocked FormButtons</div>),
 }))
 
-describe('AFPStep Component', () => {
-  const mockGoToNext = jest.fn()
-  const mockSetState = jest.fn()
-
-  const defaultFormPageProps = {
-    curStep: 1,
-    length: 5,
-    goBack: jest.fn(),
-    onStepChange: jest.fn(),
-    handleSubmit: jest.fn(),
-    goToNext: mockGoToNext,
-  }
+const mockGoToNext = jest.fn()
+const mockSetState = jest.fn()
 
-  const context = {
-    setState: mockSetState,
-    states: initialFormState,
-    formPageProps: defaultFormPageProps,
-  }
+const defaultFormPageProps = {
+  curStep: 1,
+  length: 5,
+  goBack: jest.fn(),
+  onStepChange: jest.fn(),
+  handleSubmit: jest.fn(),
+  goToNext: mockGoToNext,
+}
 
-  const mockValidateFields = jest.fn()
-  const mockClearError = jest.fn()
+const context = {
+  setState: mockSetState,
+  state: initialFormState,
+  formPageProps: defaultFormPageProps,
+}
 
-  beforeEach(() => {
-    jest.clearAllMocks()
-    ;(useErrorHandling as jest.Mock).mockReturnValue([
-      {},
-      { validateFields: mockValidateFields, clearError: mockClearError },
-    ])
-  })
+const mockValidateFields = jest.fn()
+const mockClearError = jest.fn()
 
-  const renderComponent = () => {
-    return render(
-      <FormContext.Provider value={context}>
-        <AFPStep />
-      </FormContext.Provider>
-    )
-  }
+beforeEach(() => {
+  jest.clearAllMocks()
+  ;(useErrorHandling as jest.Mock).mockReturnValue([
+    {},
+    { validateFields: mockValidateFields, clearError: mockClearError },
+  ])
+})
 
-  test('Burde rendre', () => {
+const renderComponent = () => {
+  return render(
+    <FormContext.Provider value={context}>
+      <AFPStep />
+    </FormContext.Provider>
+  )
+}
+describe('AFPStep Component', () => {
```

Comment:
Inkonsistent beskrivelser i testene. Noen ganger er assertion skrevet først, og parametre etterpå, f.eks "skal kalle handleFieldChange når radio endres" og noen ganger er det motsatt: "Dersom det tidligere er huket av NEI for AFP, Burde rendre komponenten med Nei huket av."
Bruk også gjerne stor forbokstav overalt hvis dere har begynt med det.